nep-Y.mediawiki

```diff
@@ -0,0 +1,100 @@
+<pre>
+  NEP: 25
+  Title: Smart contract transfer callback for non-fungible tokens
+  Author: Erik Zhang <erik@neo.org>, Roman Khimov <roman@nspcc.ru>
```

Comment:
I remember that onNEPXPayment was @belane 's idea, but i'm not sure

Reply:
https://github.com/neo-project/proposals/issues/108

Reply:
Worth a mention then. I was just following our NEP-11/NEP-17 here.